### ISSUE TITLE
a few static file paths make code not portable

### ISSUE THREAD

**Issue by berteh** [jwnigel/permaculture#41]
Hi nigel.
The following static paths may be problematic to deploy/use elsewhere:

```
$ grep -Ir  "/home/nigel" *
design/libs/screens/db_screen/__init__.py:data = pd.read_csv('/home/nigel/Code/permaculture/scrapers/pfaf/all_plants.csv')
design/libs/screens/db_screen/__init__.py:        db_data = pd.read_csv('/home/nigel/Code/permaculture/scrapers/pfaf/all_plants.csv')
design/env/bin/pip3.10:#!/home/nigel/code/jwnigel/permaculture/design/env/bin/python
design/env/bin/activate.fish:set -gx VIRTUAL_ENV "/home/nigel/code/jwnigel/permaculture/design/env"
design/env/bin/activate:VIRTUAL_ENV="/home/nigel/code/jwnigel/permaculture/design/env"
design/env/bin/pip:#!/home/nigel/code/jwnigel/permaculture/design/env/bin/python
design/env/bin/activate.csh:setenv VIRTUAL_ENV "/home/nigel/code/jwnigel/permaculture/design/env"
design/env/bin/pip3:#!/home/nigel/code/jwnigel/permaculture/design/env/bin/python
design/env/pyvenv.cfg:home = /home/nigel/.pyenv/versions/lewagon/bin#40 
```

**Comment by jwnigel** [comment]
Hi Bertrand,
Yes you're right. The .csv file is sort of temporary, but I think it should work if I change it to a relative path or just add the all_plants.csv to the design directory. 
As far as the virtual environment paths, would it work to delete them entirely? To be honest I no longer even use the virtual environment since I switched my entire OS to linux last week and although those environments don't exist I haven't had a problem.

I haven't tried to integrate the permapeople data yet, I noticed that many of the categories are entirely different and I've been working on adding filters and search functions to the ui with the pfaf data. Maybe you've thought about how to integrate the two sets.

Another thing I'd like to figure out is how to scrape the ENTIRE pfaf database. But I don't see how to do this without a list of the names of all 7,000 plants.
